package.json

```diff
@@ -95,6 +99,7 @@
     "realm": "12.7.1",
     "rimraf": "^2.6.1",
     "semver": "^5.5.1",
+    "sharp": "^0.33.5",
```

Comment:
just a note on this since we've tried adding the sharp package multiple times for ai highlighter and it consistently crashes on launch for macos, however i also don't see it actually being used anywhere in the PR so i'm wondering why we're adding it here?